src/components/LogView/index.js

```diff
@@ -348,8 +354,7 @@ class LogView extends React.Component {
       const windowWidth = window.innerWidth;
 
       let scrollX = window.scrollX;
-      const scrollY = window.scrollY - 45; // Account for header
-
+      const scrollY = window.scrollY; // Account for header
```

Comment:
This may have had a bad impact on vertical scrolling, can you please check and confirm?